exercises/concept-wip/arys-amazing-lasagna/.meta/config.json

```
@@ -1,10 +1,11 @@
 {
-  "authors": ["meatball133"],
+  "authors": ["meatball133", "ErikSchierboom"],
```

I think this is more fair, as you're doing all the hard work.

```suggestion
  "authors": ["meatball133"],
  "contributors": ["ErikSchierboom"],
```